src/sinks/datadog/metrics/config.rs

```diff
@@ -43,6 +43,13 @@ impl SinkBatchSettings for DatadogMetricsDefaultBatchSettings {
     const TIMEOUT_SECS: f64 = 2.0;
 }
 
+#[derive(Clone, Copy, Debug)]
+pub struct DatadogMetricsTowerRequestConfigDefaults;
+
+impl TowerRequestConfigDefaults for DatadogMetricsTowerRequestConfigDefaults {
+    const RETRY_ATTEMPTS: usize = 5;
+}
```

Comment:
Is this a change we actually want to make? It wasn't being applied previously.

Reply:
Agreed, this change is significant. I would argue for keeping it out for now. I think we need to the ability to route failed events from sinks before we lower the retry count such that it may start dropping data.

Reply:
Agree with keeping it out. Do we have context on why it should be 5?